pact/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -249,7 +249,8 @@ evaluateTerm cont handler env (Conditional c info) = case c of
   CEnforceOne str conds -> do
     case conds of
       [] ->
-        returnCEK cont handler (VError "enforce-one failure" info)
+        -- Note: this will simply be re-thrown within EnforceErrorC, so we don't need anything fancy here
+        returnCEK cont handler (VError [] (UserEnforceError "") info)
```

Comment:
Maybe we should add some context, even if it will be re-thrown, such as:

```suggestion
        returnCEK cont handler (VError [] (UserEnforceError "internal CEnforceOne error") info)
```